src/main/java/com/minecolonies/colony/Colony.java

```diff
@@ -962,4 +988,14 @@ public MaterialSystem getMaterialSystem()
     {
         return materialSystem;
     }
+
+    /**
+     * Performed when a building of this colony finished his upgrade state.
+     *
+     * @param building The upgraded building
+     * @param level The new level
+     */
+    public void onBuildingUpgradeComplete(AbstractBuilding building, int level) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this left curly brace to the beginning of next line of code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3ALeftCurlyBraceStartLineCheck)